### ISSUE TITLE
pointerup on mobile devices doesnt return features added with addObj

### ISSUE THREAD

**Issue by Strae** [OSMBuildings/OSMBuildings#171]
Hello there, i'm having some trouble trying to "select" an element.
Everythings works fine on desktop, but on mobile the `pointerup` callback always return an empry object as 'features'.

Code example:
```
let me = this;
let myMap = new OSMBuildings({
  container: 'ewo-map',
  position: { latitude: 45.472043, longitude: 9.173335 },
  zoom: 18,
  rotation: -11.4375,
  minZoom: 15,
  tilt: 45,
  maxZoom: 20,
  attribution: '© Data <a href="https://openstreetmap.org/copyright/">OpenStreetMap</a> © Map <a href="https://mapbox.com/">Mapbox</a> © 3D <a href="https://osmbuildings.org/copyright/">OSM Buildings</a>'
});

myMap.addMapTiles(
  'https://cartodb-basemaps-{s}.global.ssl.fastly.net/dark_all/{z}/{x}/{y}.png'
);

myMap.addGeoJSONTiles(
  'https://{s}.data.osmbuildings.org/0.1/anonymous/tile/{z}/{x}/{y}.json'
);

myMap.addOBJ(
  '[uyrl-to-obj-file]',
  { latitude: 45.472043, longitude: 9.173335 },
  { id: 'test', color: "#8D8D8D" }
);

myMap.on( 'pointerup', function ( t ) {
  console.log("##\t pointerup?");
  console.log( t );
});
```

On desktop, clicking the custom element give me:

```
{
  "features": [
    {
      "id": "test",
      "properties": {}
    }
  ]
}
```

On mobile, I get an empty object: `{}`

Any idea? Looks like elements added via addObj are not "seen" by `pointerup` event

**Comment by Strae** [comment]
p.s: other elements (eg. buildings)

**Comment by Strae** [comment]
sorry closed for mistake

**Comment by kekscom** [comment]
Fixed with dd64d6a
